endoreg_db/views/requirement/evaluate.py

```
@@ -0,0 +1,144 @@
+from endoreg_db.models.requirement.requirement import Requirement
+from endoreg_db.models.requirement.requirement_set import RequirementSet
+
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+@api_view(['POST'])
+def evaluate_requirements(request):
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)